README.md

```diff
@@ -8,6 +8,8 @@
 
 An ATC simulator in HTML5. Visit http://openscope.co to begin playing now!
 
+For more information on each airport, see the [airport guide](documentation/airport guide/airport-guide-directory.md).
```

Comment:
can you add a dash between airport and guide: `airport-guide`.  spaces are a no-no in urls

Reply:
done